yarn.lock

```diff
@@ -629,72 +629,72 @@
     path-browserify "^1.0.0"
     url "^0.11.0"
 
-"@sentry/core@5.29.2":
```

Comment:
Can we revert the `yarn.lock` changes? I don't think they should be included in this PR.

Reply:
This wasn't intentional.. I will revert them but I am just using whatever yarn install gives me.

Reply:
This is probably because the `package.json` changed. If you revert, then `yarn install` this should be the same.

Reply:
I am not able to make this happen without forcing the lock file from main. Could it be my node version that is doing this?

Reply:
Super weird....

Reply:
Hmm, maybe its something specific to your setup. If I run `yarn install` on your branch, it respects the lock file and does not overwrite it:
```
$ git clone -b 167/upgradable_authenticator git@github.com:gnosis/gp-v2-contracts.git TEMP
...
$ cd TEMP; yarn
...
$ git status
On branch 167/upgradable_authenticator
Your branch is up to date with 'origin/167/upgradable_authenticator'.

nothing to commit, working tree clean
```